resources/views/authors/show.blade.php

```diff
@@ -1,29 +1,36 @@
 @extends('layouts.app')
 
 @section('content')
-    <div class="container">
-        <section class="gistlog-author-block">
-            <a href="https://github.com/{{ $author->username }}" class="profile-pic">
-                <img src="{{ $author->avatarUrl }}">
+    <div class="container mx-auto gistlog__container my-8">
+        {{-- Avatar --}}
+        <div class="avatar">
+            <a href="https://github.com/{{ $author->username }}" target="_blank">
+                <img src="{{ $author->avatarUrl }}" alt="{{ $author->username }} - {{ config('app.name') }}">
             </a>
-            <h1 class="gistlog__title">{{ $author->name }}</h1>
-            <span class="gistlog__author">({{ "@" . $author->username }})</span>
-        </section>
+        </div>
+        <div class="gistlog py-8 sm:px-8">
```

Comment:
Same question here. are there any instances of `gistlog` that aren't `py-8' and `sm:px-8`? If not, then let's apply those classes to `gistlog`